gradlew

```diff
@@ -1,25 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Do we need to change this to a possibly incompatible shell? sh can be bash, but on some systems it might not be. Oh wait, is this due to gradle wrapper update?